accounts/interface.go

```diff
@@ -230,6 +230,11 @@ type Store interface {
 	AddAccountInvoice(ctx context.Context, id AccountID,
 		hash lntypes.Hash) error
 
+	// IncreaseAccountBalance increases the balance of the account with the
+	// given ID by the given amount.
+	IncreaseAccountBalance(ctx context.Context, id AccountID,
+		amount lnwire.MilliSatoshi) error
```

Comment:
I see you add the amount as `lnwire.MilliSatoshi`, so maybe we should also have a better type for the balance in `UpdateAccountBalanceAndExpiry` (see other comment)

Reply:
ah yes, great point! will update the store interface to at least use the correct types 👍